### PR TITLE
My slide1

### PR DESCRIPTION
**Pull request by Adanzeakonobi**
Microverse first project

### CODE REVIEW

File: HTML index with header [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello Microverse</title>
+  </head>
```
Comment (Deepakdanger) [inline HTML index with header]: - [ ] In `head`,Let's add link to css file. For ex:- `<link rel="stylesheet" href="index.css">`

File: index.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+header {
```
Comment (Deepakdanger) [inline index.css]: - [ ] There is no header tag in the HTML file, Kindly style the content of the `h1` tag here.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (Deepakdanger) [inline .gitignore]: - [ ] There is a .gitignore file listing `test.md` file and `node_modules` directory.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Adanzeakonobi, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] Linter should be set up correctly for this repository. Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).Let's select `HTML-css` & import all file inside him.
- [ ] Keep only one `README.md` file that contains the project description, delete another README.md file, Please don't change the file name, there is some naming convention used in git.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @Adanzeakonobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**